mantidproject/mantid
# Add polarization correction data to parameters file

**Report to:** Jos at ISIS

This PR adds polarization correction data to the offspec parameters file so that it can be used with the new polarization correction functionality in v3.13. It also adds a default for dQ/Q.

Fixes #23045

**To test:**

- Start MantidPlot and open ISIS Reflectometry interface.
- Change instrument to OFFSPEC
- From the Reflectometry menu, select `Import .tbl` and load this file (use any workspace name you like): [gui_test_table.zip](https://github.com/mantidproject/mantid/files/2218632/gui_test_table.zip)
- Go to Settings tab and click the two "Get Defaults" buttons. **Check that the `dQ/Q` field in the first row in the table gets populated with the new default of `0.03`**
- Select `ParameterFile` in PolarizationCorrection dropdown.
- Return to Runs tab and click Process.
- When reduction has finished (it will take a little while), click the "Plot the selected group" button. The results should look like this, with the red and green curves very close to zero:
![23045_results_pol_cor_offspec](https://user-images.githubusercontent.com/12895056/43068282-efbaa8fa-8e61-11e8-94f5-7baffe42527e.png)

Does this update require release notes?


---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
